dockerfiles/merge/merge.sh

```diff
@@ -3,29 +3,34 @@
 printUsage() {
    echo "Usage : "
    echo "./merge.sh SHA1 RESULTING_BRANCH"
-   echo "    SHA1: SHA1 to merge with master"
+   echo "    BRANCH: the original branch used for merge"
+   echo "    SHA1: SHA1 to merge with the branch"
    echo "    RESULTING_BRANCH : Resulting branch of the merge"
    exit 1
 }
 
-if [ "$#" -ne 2 ]; then
+if [ "$#" -ne 3 ]; then
     printUsage
 fi
 
-SHA1=$1
-RESULTING_BRANCH=$2
+BRANCH=$1
+SHA1=$2
+RESULTING_BRANCH=$3
 
 APACHE_REPO=`git remote show | grep apache || true`
 if [ -z "$APACHE_REPO" ]; then
     git remote add apache https://github.com/apache/james-project.git
 fi 
+# Getting BRANCH from apache repo
 git fetch apache
-git checkout apache/master -b master
+git checkout apache/$BRANCH -B $BRANCH
```

Comment:
why don't you default on master? This will allow to be compatible with current CI, and it seems a sane default, no?

Reply:
I hesitate on that point, will do.